utilities/utility.py

```diff
@@ -1,7 +1,18 @@
 import datetime
+import validators
 from helpers.database import db_session
 
 
+def validate_url(**kwargs):
+    """
+    Function to validate url when
+    saving an object
+    :params kwargs
+    """
+    if not validators.url(kwargs.get('image_url')):
+        raise AttributeError("the url should be valid")
```

Comment:
I would suggest using a more specific message like "The image url should be valid" or something like "Please enter a valid image url" same to the location name.